scripts/partition_manager/partition_manager.rst

```diff
@@ -718,10 +718,11 @@ Static configuration
 ********************
 
 By default, the Partition Manager dynamically places the partitions in memory.
-However, if you have a deployed product that consists of multiple images, where only a subset of the included images can be upgraded through a firmware update mechanism, the upgradable images must be statically configured.
+However, there are cases where a deployed product can consist of multiple images, and only a subset of these images can be upgraded through a firmware update mechanism.
+In such cases, the upgradable images must have partitions that are static or are matching the partition map used bootloader programmed onto the device.
```

Comment:
`are static or are matching` : It does not help if they are static (pm_static.yml) if you change the pm_static.yml in between the builds. Can I suggest that we use "and" `are static and are matching`.
`partition map used bootloader` ---> `partition map used by the bootloader`